src/sentry/rules/actions/integrations/create_ticket/utils.py

```diff
@@ -114,27 +118,36 @@ def create_issue(event: GroupEvent, futures: Sequence[RuleFuture]) -> None:
                 },
             )
             return
-        try:
-            response = installation.create_issue(data)
-        except Exception as e:
-            logger.info(
-                "%s.rule_trigger.create_ticket.failure",
-                provider,
-                extra={
-                    "rule_id": rule_id,
-                    "provider": provider,
-                    "integration_id": integration.id,
-                    "error_message": str(e),
-                    "exception_type": type(e).__name__,
-                },
-            )
-            metrics.incr(
-                f"{provider}.rule_trigger.create_ticket.failure",
-                tags={
-                    "provider": provider,
-                },
-            )
 
-            raise
+        with ProjectManagementEvent(
+            action_type=ProjectManagementActionType.CREATE_EXTERNAL_ISSUE,
+            integration=integration,
+        ).capture() as lifecycle:
+            lifecycle.add_extra("provider", provider)
```

Comment:
i was planning to add a quick little utility to remove these repeat calls  in my pr here:
https://github.com/getsentry/sentry/pull/80984/files?w=0#r1851251413.

i can update this after